config.toml

```diff
@@ -67,58 +67,59 @@ summaryLength = 70
     parent     = "section.ops"
 
 [[menu.main]]
-    name       = "Grafana"
+    name       = "Metrics"
     url        = "https://metric.sonix.network"
-    weight     = 1
+    weight     = 2
     parent     = "section.services"
 
 [[menu.main]]
     name       = "PeeringDB"
     url        = "https://peeringdb.com/ix/3607"
-    weight     = 2
+    weight     = 3
     parent     = "section.services"
 
 [[menu.main]]
     name       = "IXP Manager"
     url        = "https://manager.sonix.network/"
-    weight     = 3
+    weight     = 4
     parent     = "section.services"
 
 [[menu.main]]
-    name       = "MEMBERS"
+    name       = "SONIX Friends"
```

Comment:
I agree that it becomes a bit unclear of the purpose when using SONIX Friends instead of Members. But I also feel like Members is just as unclear, as it's essentially promoted as a service and not an association by nature. 

My goal with using that phrase was (besides that I really liked the concept and friendliness of the wording), was to make the approach more friendly and hopefully more inviting. But again, I agree that it's not really clear enough. 


I do resonate in the way that "Networks" is also quite unclear, because in the scenario where you're not perfectly clear of what you're actually looking at when visiting the website for the first time, it's not more obvious from my perspective (trying to be objective). I also fumbled around with "Connected Networks" and "Connected Peers" and even "Peers". But another problem that arises when using two words is the real estate available in the menu [INSERT GIF HERE]. 

I think it's important that even though it's a highly technical service and project at the visitors feet, that you "pratar med bönder på bönders vis" and show respect to even the unversed in the subject matter to get a chance of grasping what the project's about. 

Maybe we could have some verbal discussions on the matter?

Reply:
Sure, let's have a chat maybe tomorrow night?

Reply:
Sure, that works! I'm generally available :)

Reply:
If we compare to other IXPs in Sweden and globally, we can see that the established terminology is members, networks or connected networks. I think it will be to our disadvantage if we change this terminology. I think all or most occurrences of "Sonix Friends" across the site should be dropped. 

Examples:
* https://www.netnod.se/ix/connected-networks <- the largest in the Nordics 
* https://sthix.net/networks
* https://www.sol-ix.net/en/members/
* https://www.ams-ix.net/ams/connected-networks
* https://www.nl-ix.net/connected-networks/

Reply:
I'm not saying either I or you are right in this matter, I'm just trying to challenge the idea to ensure we try to improve, and maybe even do it better than the others. From what I can tell there's one major difference between your example businesses and SONIX, which is the fact that the others are actual businesses with the absolute goal of converting visitors to revenue, which SONIX isn't. We're purpose-driven, and I think it's a good USP to be different as well.

To be clear, I'm not saying neither you or I are right, and maybe it' just me that likes to do things differently. But I'm clear on the fact that no one should go against all logic to be different, but if there's a chance to do better, I think it's worth considering and discussing.  :)

“It’s better to be a pirate than join the navy.” --Steve Jobs